netci.groovy

```diff
@@ -54,58 +54,66 @@ build.cmd /no-deploy-extension /${configuration.toLowerCase()}""")
         // This opens the set of build steps that will be run.
         steps {
             // Build roslyn-project-system repo - we also need to set certain environment variables for building the repo with VS15 toolset.
-            batchFile("""SET VS150COMNTOOLS=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\
-SET VSSDK150Install=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\VSSDK\\
-SET VSSDKInstall=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\VSSDK\\
+            batchFile("""
+echo *** Step 1: Build Roslyn Project System ***
+SET VS150COMNTOOLS=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\Common7\\Tools\\
+SET VSSDK150Install=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
+SET VSSDKInstall=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\v15.0\\VSSDK\\
 
-build.cmd /release /skiptests""")
+build.cmd /release /skiptests
+            """)
 
             // Patch all the MSBuild xaml and targets files from the current roslyn-project-system commit into VS install.
-            batchFile("""SET VS_MSBUILD_MANAGED=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\Managed
+            batchFile("""
+echo *** Step 2: Patch the MSBuild xaml and targets ***
+SET VS_MSBUILD_MANAGED=%ProgramFiles(x86)%\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\Microsoft\\VisualStudio\\Managed
 
 mkdir backup
 xcopy /SIY "%VS_MSBUILD_MANAGED%" .\\backup\\Managed
 
 xcopy /SIY .\\src\\Targets\\*.targets "%VS_MSBUILD_MANAGED%"
 xcopy /SIY .\\bin\\Release\\Rules\\*.xaml "%VS_MSBUILD_MANAGED%"
-""")
+            """)
 
-            // Sync roslyn-internal upfront as we use the VsixExpInstaller from roslyn-internal repo to install VSIXes built from SDK repo into RoslynDev hive.
-            batchFile("""pushd %WORKSPACE%\\roslyn-internal
+            // Pull down the Open submodule of roslyn-internal as the 'Open' sources are not present until this step is executed
+            batchFile("""
```

Comment:
We should retain this step as the newer VSIX .NET  Core templates, a product of the SDK build, should install to the Roslyn hive by default.